### PR TITLE
Fix native investment

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Fixing bug with Native investment

Example of a successful claim using ETH https://rinkeby.etherscan.io/tx/0x63b9138efd234380362babc8bb2e6566750a2e9ea83ce10fe9b4862c6ad31f0f

The issue was that `parseUnits` adds X zeroes to the value passed in, but what I wanted is to remove them.
Switched back to JSBI.divide by 10**18 and now it gives us the proper amount :)

# Test

1. On rinkeby, connect your account and go to the claim page
2. Pick one account from the list below and check for the balance
3. If it has not been claimed yet, you should be able to claim it now :)
4. Trigger the claim - make sure you have the required amount of ETH + gas costs
* Your ETH claim should go through without issues

**Notes** this change might break the investor flow. I can revert that, it's on only to make sure the claiming is working

# Accounts

I filtered this list based on accounts that ONLY have ETH claims to make sure you don't need anything else (GNO/USDC)

- 0xb5434A86d09144da8854D1c40e3Cc54d78738695
- 0x9B827cc7Af79d71A2ca0f75343754935BfaF92D3
- 0x5d42D4Cd8B9D6141B1084E8eAD92e8F76b08705E
- 0x3320b6c454d0361dd3dF4912e8a06F3C945b8C60
- 0xCe0E37Bf2fC4B5c14D881f36712efB5Dcee203E8
- 0x030905cc078F3c71434fea1175615Bcc2c91FEf8
- 0xb0fE1FcEC3fE9f16c984b7A5668DF89bc7592B9D
- 0x6aBBaD8bd6Aced3649d52DFd32d2Ee04f3BE4aFB
- 0xF925FDd0de74d75Dd9C71095865c1511863092a9
- 0x26bC2B4D5DA4dc437F581AEFf490E2DdDa0aCFAa
- 0xA78266E26a6087566b17C64dE1Ed1D9533E1B36b
- 0x3877f08a4b722F48345B0e771c579fF335B533DA
- 0x1F1430C14595442c683cF3d3033a726ADecd6179
- 0xD5a0a16AA1ed668c45b29E905bafcb76C21a90BA
- 0xE6EA6d415e7Cb32bee6Db8Ad03e3749FEF23c9cd
- 0x3011862E85dDf0851f651E7d04e61e97E20a65CF
- 0x7f43A4C9E787Ba6A270e67b0D1bE77263cc47a08
- 0xCD2802122F04265791A3E4568A3f24BD1Df821a2
- 0x050f5403EB197DdE4BFE07da6e4aDE19A22db00B
- 0xe05206Ba64e59bF94c3dc3E1775Ec3FE04c33D08
- 0xC8249FB3b37fF0788462F1d20263ae373fbe68BB
- 0x01A72692ffE0A80029DE7982227A186F2787046f
- 0xDb3a676b8Eb7dBBC43b68C7Fb149f565343e850C

### CODE REVIEW

File: src/custom/pages/Claim/index.tsx [hunk 1]
```diff
@@ -209,11 +210,33 @@ export default function Claim() {
           console.log(error)
         })
     } else {
-      const inputData = [...getIndexes(freeClaims), ...selected].map((idx: number) => {
-        return userClaimData.find(({ index }) => idx === index)
-      })
+      const inputData = [...getIndexes(freeClaims), ...selected].reduce<UserClaims>((acc, idx) => {
```
Comment (W3stside) [inline src/custom/pages/Claim/index.tsx:213]: is this stuff part of the PR?

Comment (W3stside) [reply src/custom/pages/Claim/index.tsx:213]: 🦐 🤷 sheeiiittt iunno

**Review by W3stside — APPROVED** [review]

**Review by anxolin — APPROVED** [review]
👌

**Review by anxolin — COMMENTED** [review]
👌

File: src/custom/state/claim/hooks/index.ts [hunk 2]
```diff
@@ -596,7 +599,7 @@ function _getClaimValue(claim: UserClaimData, vCowAmount: string, chainId: Suppo
 
   const claimValueInAtoms = JSBI.multiply(JSBI.BigInt(vCowAmount), JSBI.BigInt(price))
 
-  return parseUnits(claimValueInAtoms.toString(), 18).toString()
+  return JSBI.divide(claimValueInAtoms, DENOMINATOR).toString()
```
[the remark this answers is not in the document]
Comment (W3stside) [reply src/custom/state/claim/hooks/index.ts:602]: `claimValueInMegaAtoms`

File: src/custom/pages/Claim/index.tsx [hunk 3]
```diff
@@ -209,11 +210,33 @@ export default function Claim() {
...
+      // TODO: this whole logic might be wrong, I'm not really checking it right now
+      setClaimStatus(ClaimStatus.ATTEMPTING)
+
+      // TODO: sort of duplicate, can be refactored later
+      // TODO: for now just for testing the claim
+      claimCallback(inputData)
+        .then((hash) => {
+          // `hash` is the hash of the claim tx. We might want to show that to the user or something like that
+          console.info(`Claim executed with the hash ${hash}`)
+          setInvestFlowStep(3) // TODO: this was the action define before. should we still call it here?
```
[the remark this answers is not in the document]
Comment (alfetopito) [reply src/custom/pages/Claim/index.tsx:233]: I'll remove this bit, there only to test the claim, will let the wiring up for another task

File: src/custom/pages/Claim/index.tsx [hunk 4]
```diff
@@ -209,11 +210,33 @@ export default function Claim() {
...
+          setClaimStatus(ClaimStatus.SUBMITTED)
+        })
+        .catch((error) => {
+          setClaimStatus(ClaimStatus.DEFAULT)
+          console.log(error)
```
[the remark this answers is not in the document]
Comment (alfetopito) [reply src/custom/pages/Claim/index.tsx:238]: It won't be there, just temporary hacky code

File: src/custom/state/claim/hooks/index.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (alfetopito) [reply src/custom/state/claim/hooks/index.ts:602]: claimValueInAtomsSquared?

[the remark this answers is not in the document]
Comment (alfetopito) [reply src/custom/state/claim/hooks/index.ts:628]: Also, it's on the docstring https://github.com/gnosis/cowswap/blob/5f9a3d1cdf9a1ae706f8f480e7099bff2d273dbe/src/custom/state/claim/hooks/index.ts#L616

**Pull request merged**